mesonpy/__init__.py

```diff
@@ -414,6 +414,44 @@ def _is_native(self, file: Union[str, pathlib.Path]) -> bool:
             return True
         return False
 
+    def _add_mesonpy_libs_to_rpath(self, path: pathlib.Path, destination: pathlib.Path) -> pathlib.Path:
+        """Adds .mesonpy.libs to the RPATH """
+        if platform.system() not in ('Linux', 'Darwin'):
+            # Internal libraries are currently unsupported on this platform
+            raise NotImplementedError(
+                "Bundling libraries in wheel is not supported "
+                f"on platform '{platform.system()}'"
+            )
+
+        if not self._is_native(path):
+            return path
+
+        # copy ELF to our working directory to avoid Meson having to regenerate the file
```

Comment:
We are operating in the install directory, not the in the build directory, therefore if the file is modified here Meson does not need to regenerate it. The installation directory, is always a temporary directory, thus the file is copied from the build directory anyway at each execution. Coping it to another temporary location is not necessary.